### ISSUE TITLE
Title of Game and Engine

### ISSUE THREAD

**Issue by Barnold1953** [VoxelGameDev/OpenVox#3]
We need to determine the name of our game and our game engine.

**Comment by Voxtric** [comment]
OpenVL? :P

**Comment by ErisianArchitect** [comment]
We could call the engine Solaris or something like that.

**Comment by Barnold1953** [comment]
Might be confused with oracle solaris.

**Comment by ErisianArchitect** [comment]
That's true. Cubos?

**Comment by Barnold1953** [comment]
We should make a discussion post on the subreddit for ideas like this.

**Comment by ErisianArchitect** [comment]
Okay, I'll make the post. But only for the name of the engine, for right now. I don't think we're far enough in to come up with a name for the game yet.

**Comment by KdotJPG** [comment]
BOOST for threading, serialization, tree structures, etc.

**Comment by DerekF** [comment]
I agree that we should use Boost, but I think you commented on the wrong issue.

**Comment by KdotJPG** [comment]
I deleted it (And I'll delete this comment saying that I deleted my comment in about a minute)
